### PR TITLE
Add Fluid Storage Monitor Cover

### PR DESCRIPTION
**Pull request by MuXiu1997**
Closed https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/8785

https://user-images.githubusercontent.com/49554020/163956675-ff8af70c-8d9f-4429-86de-a3cea27d1ead.mp4

### CODE REVIEW

File: src/main/java/gregtech/common/covers/GT_Cover_FluidStorageMonitor.java [hunk 1]
```diff
@@ -0,0 +1,415 @@
...
+
+    @Override
+    protected boolean onCoverRightClickImpl(byte aSide, int aCoverID, FluidStorageData aCoverVariable, ICoverable aTileEntity, EntityPlayer aPlayer, float aX, float aY, float aZ) {
+        if (aPlayer == null || aPlayer.worldObj == null || aPlayer.worldObj.isRemote) {
+            return false;
+        }
+        if (aPlayer.isSneaking()) {
+            return false;
+        }
+        if (aPlayer.getHeldItem() == null || !(aPlayer.getHeldItem().getItem() instanceof IFluidContainerItem)) {
```
Comment (Glease) [inline src/main/java/gregtech/common/covers/GT_Cover_FluidStorageMonitor.java:138]: This if guard make it impossible to use normal cells.

**Review by Glease — COMMENTED** [review]
Current PR doesn't seem to include a recipe. I'd suggest craft it using a GT Computer Monitor Cover and a Fluid Detector Cover inside an assembler.

**Comment by MuXiu1997** [comment]
(Making GT Computer Monitor was a pain)

File: src/main/java/gregtech/common/covers/GT_Cover_FluidStorageMonitor.java (hunk 1 above)
Comment (MuXiu1997) [reply src/main/java/gregtech/common/covers/GT_Cover_FluidStorageMonitor.java]: Already changed

**Comment by MuXiu1997** [comment]
> Current PR doesn't seem to include a recipe. I'd suggest craft it using a GT Computer Monitor Cover and a Fluid Detector Cover inside an assembler.

Added this recipe (GT Computer Monitor Cover and a Fluid Detector Cover inside an assembler) @Dream-Master

**Review by Glease — APPROVED** [review]

**Pull request merged**